Source/Libraries/PluginHost/PluginHost.csproj

```diff
@@ -1,57 +1,21 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{3F02EDA3-6949-4B30-9CDA-B56FFBA0E976}</ProjectGuid>
+    <TargetFramework>net471</TargetFramework>
     <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ExtensionHost</RootNamespace>
-    <AssemblyName>PluginHost</AssemblyName>
-    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <Deterministic>true</Deterministic>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>..\..\..\Build\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <OutputPath>..\..\..\Build\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <!-- Use Build\ instead of Build\net471\ for output -->
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <!-- System.Reflection.Metadata and System.Runtime.CompilerServices.Unsafe are conflicting dependencies https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605 -->
+    <NoWarn>NU1605</NoWarn>
```

Comment:
These conflicting dependencies are something we'll probably want to fix eventually. I know that `System.Runtime.CompilerServices.Unsafe` is the one package we can't upgrade at this point (based on our discussion in #129 ), but I'm not sure what the behavior will be when two different versions are downloaded by different parts of the same project.

Reply:
I'm honestly not sure. I may fork Ceras to resolve the CompilerServices.Unsafe conflict (it wasn't a complicated issue if memory serves me well).  Not sure about the Reflection.Metadata conflict as I don't actually know what's using that.

Reply:
I'm honestly not sure. I may fork Ceras to resolve the CompilerServices.Unsafe conflict (it wasn't a complicated issue if memory serves me well). Not sure about the Reflection.Metadata conflict as I don't actually know what's using that.

Reply:
I opened #244 to investigate this issue. For the purposes of this PR I think we should consider it out-of-scope.